src/modules/octopi/start_chroot_script

```diff
@@ -142,6 +142,12 @@ EOT
   # fetch current yq build and install to /usr/local/bin
   wget -O yq $OCTOPI_YQ_DOWNLOAD && chmod +x yq && mv yq /usr/local/bin
   
+  if [ -n "$OCTOPI_LOG_CLEAN_PATTERNS" -a -n "$OCTOPI_LOG_CLEAN_DAYS" ]; then
+    cmd="find \$HOME/.octoprint/logs -mtime +$OCTOPI_LOG_CLEAN_DAYS \\( "
+    names="$(and=; printf '%s\n' "$OCTOPI_LOG_CLEAN_PATTERNS" | tr ' ' '\n' | while read -r p; do [ -n "$p" ] || continue; printf "%s-name '%s'" "$and" "$p"; and=" -a "; done)"
+    cmd="$cmd$names \) -delete"
+    printf '@reboot %s\n@daily %s\n' "$cmd" "$cmd" | sudo -u pi crontab
```

Comment:
it'd be better to print/echo/cat this to a new file in /etc/cron.d/ or /etc/cron.daily/ if you don't mind losing the "reboot". Their syntaxes are slightly different, but it's better than blindly putting it into the UI-oriented crontab.

Reply:
/etc/cron.daily/ was my first thought but this would be run as root. On the other hand crontab(1) is the documented way to manage crontabs.

Quote from the man-page:
>       crontab  is  the program used to install, deinstall or list the tables used to
       drive the cron(8) daemon in Vixie Cron.  Each user can have their own crontab,
       and  though these are files in /var/spool/cron/crontabs, they are not intended
       to be edited directly.

Reply:
I didn't say /var/spool; If you didn't want to use root, /etc/cron.d/ uses a user specifier, eg:
```
10 3 * * * pi echo "hello from pi user"
```

The reason I know is because I forget its syntax is different :/ (and yes, this is a moot point anyhow)